File: java/src/main/java/codingdojo/Call.java
package codingdojo;

public class Call {
    private final int floor;
    private final Direction direction;


    public Call(int floor, Direction direction) {
        this.floor = floor;
        this.direction = direction;
    }

    public Direction getDirection() {
        return direction;
    }

    public int getFloor() {
        return floor;
    }
}

File: php/src/Call.php
<?php

declare(strict_types=1);

namespace Lift;

class Call
{
    public function __construct(
        private int $floor,
        private Direction $direction
    ) {
    }

    public function getDirection(): Direction
    {
        return $this->direction;
    }

    public function getFloor(): int
    {
        return $this->floor;
    }
}
